src/index.js

```diff
@@ -5,63 +5,108 @@ import {
   Route,
   Switch,
 } from 'react-router-dom';
+import { FormattedMessage } from 'react-intl';
 
+import { AppContextMenu } from '@folio/stripes/core';
 import {
   checkScope,
   CommandList,
   defaultKeyboardShortcuts,
   HasCommand,
+  NavList,
+  NavListItem,
+  NavListSection,
 } from '@folio/stripes/components';
 
+import {
+  AcqKeyboardShortcutsModal,
+  useModalToggle,
+} from '@folio/stripes-acq-components';
+
 import Main from './components/Main';
 import Settings from './settings';
 
-class Finance extends React.Component {
-  static propTypes = {
-    match: PropTypes.object,
-    showSettings: PropTypes.bool,
-    stripes: PropTypes.object,
-    history: PropTypes.object,
-  }
-
-  focusSearchField = () => {
+const Finance = ({
+  match,
+  showSettings,
+  location,
+}) => {
+  const [isOpen, toggleModal] = useModalToggle();
+  const focusSearchField = () => {
     const el = document.getElementById('input-record-search');
 
     if (el) {
       el.focus();
     }
-  }
+  };
 
-  shortcuts = [
+  const shortcuts = [
     {
       name: 'search',
-      handler: this.focusSearchField,
+      handler: focusSearchField,
+    },
+    {
+      name: 'openShortcutModal',
+      shortcut: 'mod+alt+k',
+      handler: toggleModal,
     },
   ];
 
-  render() {
-    if (this.props.showSettings) {
-      return <Settings {...this.props} />;
-    }
-
+  if (showSettings) {
     return (
+      <Settings
+        match={match}
+        location={location}
+      />
+    );
+  }
+
+  return (
+    <>
       <CommandList commands={defaultKeyboardShortcuts}>
         <HasCommand
-          commands={this.shortcuts}
+          commands={shortcuts}
           isWithinScope={checkScope}
           scope={document.body}
         >
+          <AppContextMenu>
+            {handleToggle => (
+              <NavList>
+                <NavListSection>
+                  <NavListItem
+                    id="keyboard-shortcuts-item"
+                    onClick={() => {
+                      handleToggle();
+                      toggleModal();
+                    }}
+                  >
+                    <FormattedMessage id="stripes-acq-components.appMenu.keyboardShortcuts" />
+                  </NavListItem>
+                </NavListSection>
+              </NavList>
+            )}
+          </AppContextMenu>
           <Switch>
             <Route
-              path={`${this.props.match.path}`}
-              render={() => <Main {...this.props} />}
+              path={`${match.path}`}
+              render={() => <Main match={match} />}
             />
-            <Route component={() => { this.NoMatch(); }} />
           </Switch>
         </HasCommand>
       </CommandList>
-    );
-  }
-}
+      {isOpen && (
+        <AcqKeyboardShortcutsModal
+          onClose={toggleModal}
+        />
+      )}
```

Comment:
```suggestion
      <AcqKeyboardShortcutsModal
        open={isOpen}
        onClose={toggleModal}
      />
```
See my comment here https://github.com/folio-org/stripes-acq-components/pull/494#discussion_r632631446

Reply:
Actually, `open={isOpen}` will always display modal

Reply:
Yes, you're right. `<KeyboardShortcutsModal>` has this
https://github.com/folio-org/stripes-components/blob/a804cdf3d521fe3989e8c4545ccb442811607631/lib/KeyboardShortcutsModal/KeyboardShortcutsModal.js#L64

Probably it's a mistake too...